### ISSUE TITLE
[BUG] FillAllPolygons will sometimes draw slice z=0 with current slice contour widget.

### ISSUE THREAD

**Issue by alabamagan** [wuzhuobin/blood_vessel#23]
Probably have to add something here
 InteractorSyleSmartContour2.cxx:640

```
void InteractorStyleSmartContour2::FillAllPolygons()
{
    // Only do for z-axis
    if (this->m_imageViewer->GetSliceOrientation() != 2)
        return;

    this->m_imageViewer->SetSlice(0);
        /* Generate contour for slice 0 here */
    int numOfSliceToProcess = { this->m_vesselWallPolyData.size() > this->m_lumenPolyData.size() ?
                        int(this->m_vesselWallPolyData.size()) : int(this->m_lumenPolyData.size()) };

    for (int i = 0; i < numOfSliceToProcess;i++)
    {
        this->FillPolygon();
        this->OnMouseWheelForward();
    }
    this->FillPolygon();
}
```

**Comment by alabamagan** [comment]
Also fill polygon will gets slower and slower with slice, but memory leak is not observed.

**Comment by alabamagan** [comment]
Fixed in eb36226
